client/src/main/scala/drt/client/components/FlightTableComponents.scala

```diff
@@ -59,9 +60,17 @@ object FlightTableComponents {
       <.div()
     }
 
-  def sdateLocalTimePopup(sdate: SDateLike): Unmounted[Tippy.Props, Unit, Unit] = {
-    val hhmm = f"${sdate.getHours()}%02d:${sdate.getMinutes()}%02d"
-    Tippy.describe(<.span(sdate.toLocalDateTimeString(), ^.display := "inline"), hhmm)
+  def sdateLocalTimePopup(sdateMap: Map[String, SDateLike]): Unmounted[Tippy.Props, Unit, Unit] = {
+    val sdateOpt = (sdateMap.contains("Exp"), sdateMap.contains("Expected")) match {
+      case (true, _) => sdateMap.get("Exp")
```

Comment:
This logic needs fixing and putting under test - currently it's picking a random time to display as the fallback.
It should choose from the following times in order of preference:
 - actual chox
 - estimated chox
 - touchdown
 - estimated
 - predicted
 - scheduled